### DISCUSSION TITLE
How do you pack environment?

### DISCUSSION THREAD

**Discussion by Constant206462** [deepmodeling/deepmd-kit#3088] [Q&A] [answered]
Hi. Not really deepmd related but I can't find information elsewhere.
So in the release assets there are .sh files that contain packed conda environments, right? I wanna learn how to pack my own environment into the shell (or powershell for windows) script like that.
Maybe there is a tool for that?

**Comment by njzjz** [comment, marked as answer]
https://github.com/conda/constructor

**Reply by Constant206462** [reply]
Thanks!
